src/server/game/Entities/Object/Object.cpp

```diff
@@ -2063,6 +2082,8 @@ float WorldObject::GetVisibilityRange() const
 {
     if (isActiveObject() && !ToPlayer())
         return MAX_VISIBILITY_DISTANCE;
+    else if (IsVisibilityOverride() && !ToPlayer())
+        return *m_visibilityDistanceOverride;
```

Comment:
Why not make the override have priority over `MAX_VISIBILITY_DISTANCE` for `isActiveObject()`? Why do active objects have different visibility distance in the first place???

Reply:
That's a missing cherrypick from 335. In 335 they have splitted active objects and far visible objects. So instead of isActiveObject you would have IsFarVisible here which is basically the previous version of InfiniteAOI